electrum/gui/qt/transaction_dialog.py

```diff
@@ -245,6 +251,9 @@ def insert_tx_io(
                 tcf_shortid=tcf_shortid,
                 short_id=str(txin.short_id), addr=addr, value=txin_value,
             )
+            if isinstance(txin, PartialTxInput):
+                if txin.sighash != Sighash.ALL and self.wallet.adb.is_mine(addr):
```

Comment:
txin.sighash is None by default, so as-is, this ~always warns

Reply:
ok. handled now.